modules/employee.js

```diff
@@ -0,0 +1,16 @@
+export class Employee {
+  constructor(id, salary) {
+    this.id = id;
+    this.salary = salary;
+  }
+
+  static compareSalary(employee1, employee2) {
+    if (employee1.salary > employee2.salary) {
+      return "User with id 1 has greater salary";
```

Comment:
```suggestion
      return `User with id ${employee1.id} has greater salary`;
```
Має бути ID переданого обʼєкту. Те ж саме і в інших умовах